src/components/MapSearchAutocomplete/MapSearchAutocomplete.js

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import styled from 'styled-components/macro';
+import usePlacesAutocomplete, { getGeocode, getLatLng } from 'use-places-autocomplete';
+import { Combobox, ComboboxInput, ComboboxPopover, ComboboxList, ComboboxOption } from '@reach/combobox';
+import { getCurrentPosition } from '../../utils';
+import '@reach/combobox/styles.css';
+
+export default function MapSearchAutocomplete({ setStreetAddress, setCoordinates, inputRef, defaultValue }) {
+  const {
+    ready,
+    value,
+    suggestions: { status, data },
+    clearSuggestions,
+    setValue,
+  } = usePlacesAutocomplete({ debounce: 900, defaultValue });
+
+  const getUserPosition = async () => {
+    try {
+      const position = await getCurrentPosition();
+      setCoordinates(position);
+      setValue('');
+      clearSuggestions();
+    } catch (err) {
+      alert('לא הצלחנו לאתר את המיקום.\nניתן להזין את המיקום ידנית :)');
+    }
+  };
+
+  // updates value when defaultValue changes
+  // happens on the admin page when choosing a protest
+  /*React.useEffect(() => {
+    setValue(defaultValue, false);
+    clearSuggestions();
+  }, [defaultValue, setValue, clearSuggestions]);
+
+  React.useEffect(() => {
+    if (value === '') {
+      setCoordinates(null);
+    }
+  }, [value, setCoordinates]);
+*/
+  const updateStreetAddress = (address) => setStreetAddress && setStreetAddress(address);
+
+  const handleInput = (e) => {
+    const charactersThreshold = 3;
+    const term = e.target.value;
+    const shouldFetch = term.length >= charactersThreshold;
+    setValue(term, shouldFetch);
+    updateStreetAddress(term);
+  };
+
+  const handleSelect = (address) => {
+    setValue(address, false);
+
+    getGeocode({ address })
+      .then((results) => getLatLng(results[0]))
+      .then((latLng) => {
+        const { lat, lng } = latLng;
+        setCoordinates([lat, lng]);
+        updateStreetAddress(address);
+      })
+      .catch((error) => {
+        console.log('Error: ', error);
+      });
+  };
+
+  const renderSuggestions = () => {
+    const suggestions = data.map((suggestion) => <ComboboxOption key={suggestion.place_id} value={suggestion.description} />);
+
+    return (
+      <>
+        {suggestions}
+        <li style={{ margin: '10px 10px 5px', textAlign: 'left' }}>
+          <img src="/icons/powered_google.png" alt="Powered by Google" style={{ width: 100 }} />
+        </li>
+      </>
+    );
+  };
+
+  return (
+    <Combobox style={{ position: 'absolute', zIndex: 10000, width: '100%' }} onSelect={handleSelect} aria-labelledby="demo">
+      <LocationIcon src="/icons/marker-purple.svg" alt="marker icon" />
```

Comment:
No need for `alt` tag here
```suggestion
      <LocationIcon src="/icons/marker-purple.svg" />
```